core/src/stored/label.cc

```diff
@@ -128,7 +128,7 @@ int ReadDevVolumeLabel(DeviceControlRecord* dcr)
            _("Wrong Volume mounted on device %s: Wanted %s have %s\n"),
            dev->print_name(), VolName, dev->VolHdr.VolumeName);
       if (!dev->poll && jcr->impl->label_errors++ > 100) {
-        Jmsg(jcr, M_FATAL, 0, _("Too many tries: %s"), jcr->errmsg);
+        Jmsg(jcr, M_FATAL, 0, _("Too many tries: %s\n"), jcr->errmsg);
```

Comment:
```suggestion
        Jmsg(jcr, M_FATAL, 0, _("Too many tries: %s"), jcr->errmsg);
```
`jcr->errmsg` already contains `\n`